### ISSUE TITLE
Publish shiplift 0.6 on crates.io

### ISSUE THREAD

**Issue by mandrean** [softprops/shiplift#201]
The latest version off `shiplift` on crates.io is `v0.5.0` from ~half a year ago: https://crates.io/crates/shiplift

Since then, a few things have been merged to master.

Could you publish `v0.6.0`?

**Comment by FedericoPonzi** [comment]
I just figure out this, because the `import` function is not implemented yet on `v0.5.0`...
I've temporary solved this by using:
```
shiplift =  { git = "https://github.com/softprops/shiplift.git" }
```
But it would be nice to have the updated package

**Comment by softprops** [comment]
Published 0.6.0 with all recent changes. 0.7.0 will likely target newer async deps
